src/app/src/main/kotlin/org/imperial/mrc/hint/service/ProjectAndVersionService.kt

```diff
@@ -0,0 +1,44 @@
+package org.imperial.mrc.hint.service
+
+import org.imperial.mrc.hint.AppProperties
+import org.imperial.mrc.hint.db.ProjectRepository
+import org.imperial.mrc.hint.db.VersionRepository
+import org.imperial.mrc.hint.exceptions.UserException
+import org.imperial.mrc.hint.logging.GenericLoggerImpl
+import org.imperial.mrc.hint.logic.UserLogic
+import org.imperial.mrc.hint.security.Session
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+interface ProjectVersionService
+{
+    fun cloneProjectToUser(projectId: Int, emails: List<String>)
+}
+
+@Service
+class ProjectAndVersionService (
```

Comment:
I'm a bit baffled by the respective names of the interface and implementation!
Does it really need to be an interface? There's only ever going to be one implementation, and we can still mock the class for testing. 
I think `ProjectService` would be fine personally!